tests/componentTests/podcasts/test.httpRequests.spec.js

```diff
@@ -0,0 +1,91 @@
+const supertest = require('supertest')
+const express = require('express')
+const router = require('../../../routes')
+const { badFieldsMockPodcast, mockPodcast, badMockUpdate, mockNewPodcast } = require('./mockPodcast.js')
+const bodyParser = require('body-parser')
+const app = express()
+
+app.use(router)
+app.use(bodyParser.urlencoded({ extended: false }))
+app.use(bodyParser.json())
+
+jest.mock('../../../models/fileModel', () => ({
+  getItem: (id) => id === 1132 ? mockPodcast : null,
+  deleteItem: (id) => Promise.resolve(),
+  updateItem: (id, updatedPodcast) => Promise.resolve(),
+  saveItem: (podcast) => Promise.resolve(),
+  getMaxItem: () => Promise.resolve()
+}))
+
+describe('HTTP requests component test', () => {
+  describe('`POST` method Succeeds to post new podcast', () => {
+    it('calls post /podcast/new, should save mockNewPodcast to database', async () => {
+      await supertest(app)
+        .post('/podcast/new')
+        .send(JSON.parse(JSON.stringify(mockNewPodcast)))
+        .expect(200, 'Podcast has been successfully added')
+    })
+  })
+  describe('`POST` method Fails to post podcast request', () => {
+    it('calls post /podcast/new, should return 500 error', async () => {
```

Comment:
Post requests should contain the following tests:
1. When a field that doesn't exist passed it should return 400
2. When passing a field in the wrong format it should return 400
3. when a field is missing it should return 400